src/components/DateTime/DateTimePicker.module.css

```diff
@@ -0,0 +1,12 @@
+.input {
```

Comment:
Hey—Eric here, nice to meet you! Can you please use tailwind instead of a css file? Thanks!